### PR TITLE
Make available envs displayed in env.properties

### PR DESCRIPTION
**Pull request by michgauz**

### CODE REVIEW

File: kinta-lib/src/main/kotlin/com/dailymotion/kinta/KintaEnv.kt [hunk 1]
```diff
@@ -1,31 +1,36 @@
...
-    const val GOOGLE_CLOUD_STORAGE_JSON = "GOOGLE_CLOUD_STORAGE_JSON"
-    const val JIRA_USERNAME = "JIRA_USERNAME"
-    const val JIRA_PASSWORD = "JIRA_PASSWORD"
-    const val BITRISE_PERSONAL_TOKEN = "BITRISE_PERSONAL_TOKEN"
-    const val JIRA_URL = "JIRA_URL"
-
-    const val APPLE_PASSWORD = "APPLE_PASSWORD"
-    const val APPLE_USERNAME = "APPLE_USERNAME"
+
+    enum class Env {
```
Comment (martinbonnin) [inline kinta-lib/src/main/kotlin/com/dailymotion/kinta/KintaEnv.kt]: what about ?
```suggestion
    enum class Var {
```

This way, the callsite will look like `KintaEnv.get(KintaEnv.Var. APPLE_PASSWORD)`

File: kinta-lib/src/main/kotlin/com/dailymotion/kinta/KintaEnv.kt [hunk 2]
```diff
@@ -35,6 +40,8 @@ object KintaEnv {
         return System.getenv(key)
     }
 
+    fun getOrFail(env: Env) = getOrFail(env.name)
+
     fun getOrFail(key: String): String {
```
Comment (martinbonnin) [inline kinta-lib/src/main/kotlin/com/dailymotion/kinta/KintaEnv.kt]: Do we still need the `String` versions ?

Comment (michgauz) [reply kinta-lib/src/main/kotlin/com/dailymotion/kinta/KintaEnv.kt]: I have kept it for users who want to add their owes kinta env.
For exemple, our CI_BACKEND_TOKEN

File: kinta-lib/src/main/kotlin/com/dailymotion/kinta/KintaEnv.kt (hunk 1 above)
Comment (michgauz) [reply kinta-lib/src/main/kotlin/com/dailymotion/kinta/KintaEnv.kt]: 👍

**Pull request merged**